dev-docs/bidders/deepintent.md

```diff
@@ -20,6 +21,11 @@ usp_supported: true
 | `user`        | optional | user params according to IAB standards | `'{"gender":"F"}''`| `object` |
 | `pos`         | optional | ad position as per IAB standards       | `1`                | `number` |
 
+
+
+### Prebid Server Note:
+While using prebid server mode, "banner.h" and "banner.w" should be passed. Support for the format is not present at the moment. 
```

Comment:
@sourabhg - I'm afraid this note is going to confuse people.

1. banner.h and banner.w isn't something most people have control over -- that part of the OpenRTB is created by the pbsBidAdapter, the Prebid SDK, or is part of a stored request. I think you should remove this note because publishers reading it don't have any actions to take.

2. "Support for the format is not present at the moment" -- what does this mean?  Does it mean video and native aren't supported?

Reply:
suport for Imp.banner.Format array is not present at the moment. I think you are right people will get confused by this and I will remove this note.